### PR TITLE
TAXII - Added authentication using Certificate key and text file.

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] ~~In Progress - customer validation~~ READY

## Related Issues
fixes: https://github.com/demisto/etc/issues/22911

## Minimum version of Demisto
- [x] 5.5.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Legacy/Integrations/FeedTAXII/FeedTAXII.yml [hunk 1]
```diff
@@ -1,28 +1,24 @@
...
+- additionalinfo: Indicators from this integration instance will be marked with this reputation
   display: Indicator Reputation
   name: feedReputation
   options:
   - None
   - Good
   - Suspicious
   - Bad
-  required: false
+  required: False
```
Comment (yuvalbenshalom) [inline Packs/Legacy/Integrations/FeedTAXII/FeedTAXII.yml:19]:
```suggestion
  required: false
```

File: Packs/Legacy/Integrations/FeedTAXII/FeedTAXII.yml [hunk 2]
```diff
@@ -32,9 +28,9 @@ configuration:
   - D - Not usually reliable
   - E - Unreliable
   - F - Reliability cannot be judged
-  required: true
+  required: True
```
Comment (yuvalbenshalom) [inline Packs/Legacy/Integrations/FeedTAXII/FeedTAXII.yml:31]:
```suggestion
  required: true
```

File: Packs/Legacy/Integrations/FeedTAXII/FeedTAXII.py [hunk 3]
```diff
@@ -517,6 +521,23 @@ def __init__(self, insecure: bool = True, polling_timeout: int = 20, initial_int
                 self.username = credentials.get('identifier', None)
                 self.password = credentials.get('password', None)
 
+        if cert_text and key_text:
```
Comment (yuvalbenshalom) [inline Packs/Legacy/Integrations/FeedTAXII/FeedTAXII.py:524]: Can this work if only one of them is inserted in some cases?
If not, we should return an error in the test stating that 2 of the params should be filled, or none of them.

File: Packs/Legacy/Integrations/FeedTAXII/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+Added authentication using Certificate key and text file.
```
Comment (amshamah419) [inline Packs/Legacy/Integrations/FeedTAXII/CHANGELOG.md:2]: Added authentication using certificate key and text file.

**Pull request merged**